examples/react-load/preact/package.json

```diff
@@ -37,11 +37,11 @@
     "sirv-cli": "1.0.3"
   },
   "dependencies": {
-    "@opentelemetry/context-zone": "^0.19.0",
-    "@opentelemetry/exporter-collector": "^0.19.0",
+    "@opentelemetry/context-zone": "^0.20.0",
+    "@opentelemetry/exporter-collector": "^0.20.0",
     "@opentelemetry/plugin-react-load": "^0.15.0",
```

Comment:
almost all examples point to an outdated version of itself. But maybe fix this in a follup

Reply:
Will do in follow up good catch